### PR TITLE
modified chatvectordb with fixes that pass the system messages. 

### PR DESCRIPTION
**Pull request by OmriNach**

### CODE REVIEW

**Comment by OmriNach** [comment]
#442

File: langchain/src/chains/question_answering/load.ts [hunk 1]
```diff
@@ -1,91 +1,48 @@
...
-) => {
-  const {
-    prompt = DEFAULT_QA_PROMPT,
-    combineMapPrompt = DEFAULT_COMBINE_QA_PROMPT,
-    combinePrompt = COMBINE_PROMPT,
-    type = "stuff",
-  } = params;
-  if (type === "stuff") {
-    const llmChain = new LLMChain({ prompt, llm });
+import { COMBINE_PROMPT, DEFAULT_COMBINE_QA_PROMPT, COMBINE_PROMPT_SELECTOR, COMBINE_QA_PROMPT_SELECTOR, } from "./map_reduce_prompts.js";
```
Comment (nfcampos) [inline langchain/src/chains/question_answering/load.ts:4]: Looks like you're changing a lot of unrelated lines? If you could clean up the diff then we can review your changes
